daprdocs/data/components/state_stores/generic.yaml

```
@@ -45,7 +45,7 @@
 - component: etcd
   link: setup-etcd
   state: Beta
-  version: v1
+  version: v2
   since: "1.11"
```

Should change this to indicate when this version was available from
```suggestion
  since: "1.12"
```